### PR TITLE
Animation fixes

### PR DESCRIPTION
**Pull request by tpltnt**
Hi @ofZach 

please take a look at my changes to the "animation" chapter.
Thank you,
tpltnt

### CODE REVIEW

**Comment by ofZach** [comment]
amazing, thanks! - checking.  will need a few days to evaluate

**Comment by tpltnt** [comment]
@ofZach how far did you get?

File: chapters/animation/chapter.md [hunk 1]
```diff
@@ -96,50 +121,196 @@ You will also see objects that have basic functionality and internal variables.
...
 
-To add some detail if we are 0 pct of the way from A to B, we calculate 
+To add some detail if we are 0 pct of the way from A to B, we calculate
 
 ```cpp
 ((1-0) * A) + (0 * B)
 ```
 
-which simplifies to `(1*A + 0*B)` or A. If we are 25 percent of the way, it looks like 
+which simplifies to `(1*A + 0*B)` or A. If we are 25 percent of the way, it looks like
```
Comment (tpltnt) [inline chapters/animation/chapter.md]: also pointed out in #151

**Comment by tpltnt** [comment]
ping @ofZach

**Comment by ofZach** [comment]
thanks!

File: chapters/animation/chapter.md [hunk 2]
```diff
@@ -65,29 +90,29 @@ void testApp::draw(){
...
 
 ```cpp
 void testApp::update(){
-    xPos += 0.5 * (30.0 / ofGetFrameRate());
+    xPos += ofGetLastFrameTime() * speed;
 }
 ```
 
-If ofGetFrameRate() returns 30, we multiply 0.5 by 1, if ofGetFrameRate() returns 60, we multiply it by 1/2, so although we are animating twice as fast, we take half sized steps, therefore effectively moving at the same speed regardless of frame rate.  Frame rate independence is fairly important to think about once you get the hang of things, since as observers of animation, we really do feel objects speeding up or slowwing down even slightly, but in this chapter I will skip it for the sake of simplicity in the code. 
+If `ofGetFrameRate()` returns 30, we multiply 0.5 by 1, if `ofGetFrameRate()` returns 60, we multiply it by 1/2, so although we are animating twice as fast, we take half sized steps, therefore effectively moving at the same speed regardless of frame rate.  Frame rate independence is fairly important to think about once you get the hang of things. As observers of animation, we really do feel objects speeding up or slowing down even slightly, but in this chapter I will skip it for the sake of simplicity in the code.
```
Comment (arturoc) [inline chapters/animation/chapter.md]: this paragraph still explains the old example instead of using ofGetLastFrameTime

Comment (tpltnt) [inline chapters/animation/chapter.md]: Good catch @arturoc, i (hopefully) fixed it in #168.

**Pull request merged**